docs/api/crash-reporter.md

```diff
@@ -96,7 +96,7 @@ them will get reported without `companyName`, `productName` or any of the `extra
 Returns [`CrashReport`](structures/crash-report.md):
 
 Returns the date and ID of the last crash report. If no crash reports have been
-sent or the crash reporter has not been started, `null` is returned.
+sent (i.e a crash was triggered with `process.crash()` and crash reports are present in the `crashDirectory`, but have not been explicitly uploaded) or the crash reporter has not been started, `null` is returned.
```

Comment:
I think mentioning `process.crash()` here might throw people a bit off the trail, maybe just saying something along the lines of "only crash reports that have been uploaded will be returned, even if a crash report is present on disk it will not be returned until it is uploaded", just slightly better than that 😆